### PR TITLE
Aoc/659 correct snippet key text

### PR DESCRIPTION
**Pull request by chengoconnell**
## Types of changes

- [x] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly. (PR: #)
- [ ] All new and existing tests are passing.

### CODE REVIEW

File: packages/slice-machine/lib/builders/common/Zone/Card/components/Hints/Renderers/react.tsx [hunk 1]
```diff
@@ -110,7 +106,7 @@ const toReact: React.FC<{
   isRepeatable: boolean;
 }> = ({ Widgets, item, typeName, renderHintBase, isRepeatable }) => {
   const hintBase = renderHintBase({ item });
-
+  console.log({ Widgets, item, typeName, renderHintBase, isRepeatable });
```
Comment (MarcMcIntosh) [inline packages/slice-machine/lib/builders/common/Zone/Card/components/Hints/Renderers/react.tsx]: console.log

File: packages/slice-machine/lib/builders/common/Zone/Card/components/Hints/Renderers/react.tsx [hunk 2]
```diff
@@ -95,11 +95,7 @@ const codeByWidgetType = (
       useKey ? appendKey("color") : ""
     } style={{ color: ${fieldText} }}>Some Text</span>`,
 
-  [Widgets.Text?.TYPE_NAME]: (
-    fieldText: string,
-    useKey?: boolean
-  ) => `/* import { PrismicText } from '@prismicio/react' */
-<PrismicText ${useKey ? appendKey("text") : ""} field={${fieldText}} />`,
+  [Widgets.Text?.TYPE_NAME]: (fieldText: string) => `{${fieldText}}`,
```
Comment (MarcMcIntosh) [inline packages/slice-machine/lib/builders/common/Zone/Card/components/Hints/Renderers/react.tsx]: `ceateDefaultField` would work here too?

Comment (chengoconnell) [reply packages/slice-machine/lib/builders/common/Zone/Card/components/Hints/Renderers/react.tsx]: Screenshot 2022-06-21 at 14 58 39
createDefaultField would gives us this ⬆️ . According to the slack messages we want this? ⬇️
Screenshot 2022-06-21 at 14 59 25

Comment (MarcMcIntosh) [reply packages/slice-machine/lib/builders/common/Zone/Card/components/Hints/Renderers/react.tsx]: `{slice.primary.callToAction}` will work, but I think we should keep it uniform with all the other ones that would also work with out the `<span>` tag :)

Comment (chengoconnell) [reply packages/slice-machine/lib/builders/common/Zone/Card/components/Hints/Renderers/react.tsx]: Sure will change! :)

**Review by sintinilorenzo — APPROVED** [review]

**Pull request merged**